src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -543,7 +543,7 @@ void CouplingSchemeConfiguration::addTagResidualRelativeConvergenceMeasure(
                                XMLTag::OCCUR_ARBITRARY);
   tagConvergenceMeasure.setDocumentation(
       "Residual relative convergence criterion based on the relative two-norm differences of data values between iterations.\n"
-      "\\$$\\frac{\\left\\lVert H(x^k) - x^k \\right\\rVert_2}{\\left\\lVert H(x^{k-1}) - x^{k-1} \\right\\rVert_2} < \\text{limit}\\$$");
+      "\\$$\\frac{\\left\\lVert H(x^k) - x^k \\right\\rVert_2}{\\left\\lVert H(x^0) - x^0 \\right\\rVert_2} < \\text{limit}\\$$");
```

Comment:
May I ask you to also update the documentation sentence: "values between iterations" doesn't fit to the formula IMO

Reply:
I tried to come up with a clearer description, see the new commit! (It's arguably clearer than those for the other convergence criteria now, but I don't want to start working on those as well :smile:)